tools/rollup-es3/src/ImportCheck.ts

```diff
@@ -13,8 +13,23 @@ export function importCheck(options:IImportCheckRollupOptions = {}) {
         keywords: []
     };
 
+    // Don't allow importing from folders from a package
+    checkOptions.keywords.push({
```

Comment:
Just extra build validations to ensure imports and usages of EnumMaps are done correctly.